### ISSUE TITLE
Unable to install p2g_glue on macOS Sierra

### ISSUE THREAD

**Issue by andyprata** [davidh-ssec/polar2grid#125]
Hi,

I have recently upgraded my Mac to Sierra and my polar2grid installation has broken (i.e. my p2g_glue command is no longer recognised).  Below is the command I'm trying to run and the associated error.

```
andyprata$ pip install --user --trusted-host larch.ssec.wisc.edu --extra-index-url http://larch.ssec.wisc.edu/simple/ polar2grid[awips,gtiff,remap,utils]
Requirement already satisfied: polar2grid[awips,gtiff,remap,utils] in ./anaconda/lib/python2.7/site-packages/polar2grid-2.0.1-py2.7-macosx-10.6-x86_64.egg
Requirement already satisfied: setuptools in ./anaconda/lib/python2.7/site-packages (from polar2grid[awips,gtiff,remap,utils])
Requirement already satisfied: numpy in ./anaconda/lib/python2.7/site-packages (from polar2grid[awips,gtiff,remap,utils])
Collecting polar2grid.core[all] (from polar2grid[awips,gtiff,remap,utils])
  Could not find a version that satisfies the requirement polar2grid.core[all] (from polar2grid[awips,gtiff,remap,utils]) (from versions: )
No matching distribution found for polar2grid.core[all] (from polar2grid[awips,gtiff,remap,utils])
```
Any ideas about what's going wrong?

Thanks,
Andrew

**Comment by davidh-ssec** [comment]
Hi Andrew,

Sorry for the delay on responding (been out sick). This is likely due to something I've screwed up since I don't normally install Polar2Grid this way, but it is also not really supported. One thing you could try is installing `polar2grid.core` directly:

    pip install http://larch.ssec.wisc.edu/eggs/repos/polar2grid.core/polar2grid.core-2.0.0.tar.gz

And then try your command again. If that doesn't work then I can provide you instructions for installing from the code on github. Note that with recent changes to Polar2Grid (soon to be released in version 2.1) and planned changes, how Polar2Grid is installed will change a lot in the future.

**Comment by andyprata** [comment]
Hi David,

Thank you for responding to my issue and sorry to hear you've been out sick.  I was able to get it all working again after downloading and installing the code from github.
